docs/graphs/getting-started-with-graphs.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Getting Started with Graphs
+sidebar_position: 10
+---
+
+In this getting started guide, you'll explore how to work with graphs in Macrometa by establishing relationships between collections, enabling you to run complex queries on airline flight data.
+
+## About Graphs
+
+Graphs provide a mechanism to model your data and perform sophisticated queries across interconnected documents. A graph is composed of _vertices_ and _edges_.
+
+- Vertices are stored in collections and linked by an edge document. Vertices can be either a document or an edge.
```

Comment:
"Vertices can be either a document or an edge"
A vertex is always a document and never an edge.

Reply:
And this is why we have SMEs. Nice catch! :)